dynamics/src/cgParametricMomentum.cpp

```diff
@@ -216,8 +216,12 @@ void CGParametricMomentum<CG>::mEVPStep(const VPParameters& params, const size_t
 
     // Update the stresses according to the mEVP model
 
-    Nextsim::mEVP::StressUpdateHighOrder(
-        params, pmap, smesh, S11, S12, S22, E11, E12, E22, H, A, alpha, beta);
+    std::cerr << "Fatal: Stressupdatehigherorder requires ParametricMomnetumMap with proper "
+                 "template parameter. See headerfile and use Kernel-infrastructure"
+              << std::endl;
```

Comment:
"CGParametricMomentum::mEVPStep(): Fatal: StressUpdateHighOrder requires ParametricMomentumMap with proper template parameter. See header file and use kernel infrastructure."